TMDbLibTests/TMDbLibTests.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\_Imports\Test.targets" />
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
as you are now testing net-standard2.0 you should add a dotnet framework as a TargetFramework (like 4.6.2) to actually test net-standard2.0